deployment/cdk/opensearch-service-migration/README.md

```diff
@@ -1,184 +1,187 @@
 
+1- Install the **CDK CLI** tool, if you haven't already, by running:
+```shell
+npm install -g aws-cdk
 ```
-cdk bootstrap
+
+2- **Bootstrap CDK**: if you have not run CDK previously in the configured region of you account, it is necessary to run the following command from the `opensearch-service-migration` directory to set up a small CloudFormation stack of resources that CDK needs to function within your account
+
+```shell
+# Execute from the deployment/cdk/opensearch-service-migration directory
+cdk bootstrap --c contextId=demo-deploy
 ```
 
 Further CDK documentation [here](https://docs.aws.amazon.com/cdk/v2/guide/cli.html)
 
-### Deploying your CDK
-Before deploying your CDK you should fill in any desired context parameters that will dictate the composition of your OpenSearch Service Domain
+## Deploying the CDK
+This project uses CDK context parameters to configure deployments. These context values will dictate the composition of your stacks as well as which stacks get deployed.
+
+The full list of available configuration options for this project are listed [here](./options.md). Each option can be provided as an empty string `""` or simply not included, and in each of these 'empty' cases the option will use the project default value (if it exists) or CloudFormation's default value.
 
-This can be accomplished by providing these options in a `cdk.context.json` file and then deploying like so:
+A set of demo context values (using the `demo-deploy` label) has been set in the `cdk.context.json` located in this directory, which can be customized or used as is for a quickstart demo solution.
+
+This demo solution can be deployed with the following command:
+```shell
+cdk deploy "*" --c contextId=demo-deploy --require-approval never --concurrency 3
 ```
-cdk deploy "*"
+
+Additionally, another context block in the `cdk.context.json` could be created with say the label `uat-deploy` with its own custom context configuration and be deployed with the command:
+```shell
+cdk deploy "*" --c contextId=uat-deploy --require-approval never --concurrency 3
 ```
+**Note**: Separate deployments within the same account and region should use unique `stage` context values to avoid resource naming conflicts when deploying (**Except** in the multiple replay scenario stated [here](#how-to-run-multiple-replayer-scenarios)) 
 
-Or by passing the context options you want to change as options in the CDK CLI
+Stacks can also be redeployed individually, with any required stacks also being deployed initially, e.g. the following command would deploy the migration-console stack
+```shell
+cdk deploy "migration-console" --c contextId=demo-deploy
 ```
-cdk deploy "*" --c domainName="os-service-domain" --c engineVersion="OS_1_3_6" --c dataNodeType="r6g.large.search" --c dataNodeCount=1
+
+To get a list of all the available stack ids that can be deployed/redeployed for a particular `contextId`:
+```shell
+cdk synth "*" --c contextId=demo-deploy
```

Comment:
Nit - to list available stacks, advise `cdk ls` instead. `Cdk synth` is a dry-run to generate the CloudFormation templates that will be used by `cdk deploy`

Reply:
Agreed have changed now